account/recovery.go

```diff
@@ -17,6 +20,69 @@ var (
 	DefaultAccountKeyWindow uint32 = 500
 )
 
+// auctioneerData contains the auctioneers' information needed to recovery
+// the user's accounts without interacting with the server.
+//
+// Note: Each environment has its own values.
+type auctioneerData struct {
+	// Initial value for the batch key.
+	InitialBatchKey *btcec.PublicKey
+
+	// Auctioneer public key.
+	PubKey *btcec.PublicKey
+
+	// Block height for the first batch ever.
+	FirstBatchBlock uint32
+}
+
+// getAuctioneerData returns the auctioner data for a given environment.
+func getAuctioneerData(network string) (*auctioneerData, error) {
+	var auctioneerKey string
+	var batchKey string
+	var fstBlock uint32
+
+	switch network {
+	case "mainnet":
+		auctioneerKey = "034d1fd68b5cd92f694766aa8b3f90131613d1b731" +
+			"caf03be5d2d34c333163c606"
+		batchKey = "02824d0cbac65e01712124c50ff2cc74ce22851d7b444c1" +
+			"bf2ae66afefb8eaf27f"
+		fstBlock = 648168
+	case "testnet":
+		// TODO: add default values for testnet.
+	case "regtest":
```

Comment:
I wonder if instead we should make it possible to overwrite these values through the CLI/gRPC instead? So there would be a field called `auctioneer_key` and `first_block` in the `RecoverAccounts` RPC that could be left empty to use default values but set for regtest.